ResultPicker.js

```diff
@@ -1,36 +1,49 @@
-document.selectedResultId=0
+// modified version of https://superuser.com/a/1235058/69589
 
-function selectResult(newId){
-    els = document.querySelectorAll("div.r h3")
-    if(newId < 0 || newId >= els.length)
-        return  //Could modify for page nav...?
-    rp = document.getElementById("result-pointer")
-    if(rp != null){
-        rp.remove()
-    }
-    document.selectedResultId=newId
-    el = els[newId]
-    lnk = el.parentElement
-    el.innerHTML = "<div id=\"result-pointer\" style=\"position:absolute;left:-15px;\">&gt;</div>" + el.innerHTML
-    lnk.focus()
+// find all the search results
+const querySelector = '.rc > div.yuRUbf';
+
+document.selectedResultId = 0;
+function selectResult(newId) {
```

Comment:
other than the `querySelector` the only stuff that changed in this function is whitespace and the addition of `{` and `}` and `;`